### ISSUE TITLE
Bugs Release

### ISSUE THREAD

**Issue by EPajares** [goat-community/goat#3312]

**Comment by EPajares** [comment]
Check download for layers in a shared project as editor

**Comment by CyrineKamoun** [comment]
As an editor, I have to first create a folder to be able to create a project. There is no automatic initial folder where the project can be linked to.

**Comment by CyrineKamoun** [comment]
Adding explanation on the dcumentation about project sharing or having a warning message when deleting a project in a personal document that is share with a team that it will also be deleted to other members

**Comment by CyrineKamoun** [comment]
Calculation station nearby does not work

**Comment by gokulgpn** [comment]
I am unable to view the data after filter is being applied

**Comment by gokulgpn** [comment]
**Filter**: I wanted to filter some polygons that has attribute values  **NULL**. Is there any way to do that?

**Comment by gokulgpn** [comment]
**Basemap issue** I created a new project with dark base map, but when I go to the the home and come back to the project the basemap becomes high fidelity by default

**Comment by CyrineKamoun** [comment]
I shared a project in a team with  editor authorisation. It worked (but i can only see the update they add when refreshing the project page). Then i wanted to change the authorisation to viewer: there is an error and it does not work (even though all team members do not have the project open)

**Comment by gokulgpn** [comment]
**Error Upload** Getting error when uploading csv file

**Comment by CyrineKamoun** [comment]
A viewer was able to upload a document (but he can't do anything else)

**Comment by gokulgpn** [comment]
**Filter Spatial Expression** I think the boundary section in the spatial expression is not working. Also I cannot find the spatial expression in the GOAT DOCS

**Comment by nihar1024** [comment]
I'm unable to open the shared "test" project.  Seems to be associated with the user project association:

<img width="778" alt="Screenshot 2024-09-18 at 9 33 36 PM" src="https://github.com/user-attachments/assets/29af9b23-4214-4995-b76c-ef94145bc848">

**Comment by nihar1024** [comment]
> **Error Upload** Getting error when uploading csv, gpkg files

Could you send me the files on Teams? I'll try to debug the issue.

**Comment by majkshkurti** [comment]
> I still get the `Bad Gateway` response occasionally. Please upvote this comment if you see it too.

I think this is because I was deploying the client on that moment. It shouldn't happen in prod as it we will have more than 1 pod

**Comment by nihar1024** [comment]
> > I still get the `Bad Gateway` response occasionally. Please upvote this comment if you see it too.
> 
> I think it happened because I was deploying the client on that moment. It shouldn't happen in prod we will have more than 1 pod

Ah ok, makes sense.

**Comment by EPajares** [comment]
Revise delete user logic

**Comment by EPajares** [comment]
> The language/display is linked with the organization setting not with the user account

I cannot really reproduce this. Anyone else had this problem?

**Comment by EPajares** [comment]
Check project permissions for users joining existing orgs and teams

**Comment by EPajares** [comment]
> I am unable to view the data after filter is being applied

Which dataset ?

**Comment by EPajares** [comment]
> A viewer was able to upload a document (but he can't do anything else)

UI for Viewer is not finished yet

**Comment by EPajares** [comment]
Failing to share project/layer with organization after having shared with a team

**Comment by EPajares** [comment]
Disable deleting for content from team and org

![image](https://github.com/user-attachments/assets/352510ef-9a44-4813-8a58-5ab52dd6f4a4)

**Comment by nihar1024** [comment]
> > The language/display is linked with the organization setting not with the user account
> 
> I cannot really reproduce this. Anyone else had this problem?

I think this might be due to an issue with the cookie? When I change the language, the new language only applies to the settings page but not any of the other pages.

**Comment by nihar1024** [comment]
> Erro while downlowding databse (csv or excel) that I imported earlier in a project

This seems to work for me. Would you please be able to send me one of the files you're testing with (on Teams) and I can take a look?

**Comment by CyrineKamoun** [comment]
when sending invitations to new member there is the option "admin" But when i send an invitiation with admin authorisation and then the person creating the new account need to create an organization too and is not automatically linked to the previous organizaiton from which he got the invitation

**Comment by gokulgpn** [comment]
**Project Share** When sharing the project with others, I tried to upload a dataset and my team member can see the dataset in the project, but cannot see the dataset in the 'Dataset'. Though my team member can see the dataset in the project( she could do every analysis) , but cannot download the metadata from the project

**Comment by CyrineKamoun** [comment]
when deleting a scenario, the changes that where in the scenario does not disappear automatically but the page need to be refreshed so that the change are not visible anymore in the map

**Comment by gokulgpn** [comment]
**Project Share** My team member shared me a project and I worked on the shared project. However I cannot see the project in the **'Recent Project'** section. I have to go inside **Projects** every time

**Comment by CyrineKamoun** [comment]
When you invite a member to your organisation, he create a project and shares it to the organization, everyone has access to it. BUT if he is removed from the organisation, the project he uploaded disappear to all organisation members

**Comment by CyrineKamoun** [comment]
> **Project Share** When sharing the project with others, I tried to upload a dataset and my team member can see the dataset in the project, but cannot see the dataset in the 'Dataset'. Though my team member can see the dataset in the project( she could do every analysis) , but cannot download the metadata from the project

If i share a dataset to the organization directly, other members can see the metadata but can not download the dataset

**Comment by EPajares** [comment]
> When you invite a member to your organisation, he create a project and shares it to the organization, everyone has access to it. BUT if he is removed from the organisation, the project he uploaded disappear to all organisation members

I guess this is what the user would expect right? As it is his project

**Comment by gokulgpn** [comment]
**Catalog** I imported the dataset from data catalog, changed styling and then saved the style as default. So this affected the main data in the catalog also. Do you think th estyle in the main data should not affect, since all the descriptions are based on the styling?
![Screenshot 2024-09-19 101429](https://github.com/user-attachments/assets/603942a0-639f-47d6-a8f8-77e740f2ba8e)

**Comment by CyrineKamoun** [comment]
> > When you invite a member to your organisation, he create a project and shares it to the organization, everyone has access to it. BUT if he is removed from the organisation, the project he uploaded disappear to all organisation members
> 
> I guess this is what the user would expect right? As it is his project

Maybe the user looks for that if it is his own project, but if the project is done by two members. Let's say member A creates a project and shares it with member B. Than member B works on it 50% of his time. After some time and before ending the project, member A leaves the compagny: than member B loses all his work on that project

**Comment by gokulgpn** [comment]
> **Catalog** I imported the dataset from data catalog, changed styling and then saved the style as default. So this affected the main data in the catalog also. Do you think the style in the main data should not get affected, since all the descriptions are based on the styling? ![Screenshot 2024-09-19 101429](https://private-user-images.githubusercontent.com/99094073/368904569-603942a0-639f-47d6-a8f8-77e740f2ba8e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY3MzQ0MzEsIm5iZiI6MTcyNjczNDEzMSwicGF0aCI6Ii85OTA5NDA3My8zNjg5MDQ1NjktNjAzOTQyYTAtNjM5Zi00N2Q2LWE4ZjgtNzdlNzQwZjJiYThlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE5VDA4MjIxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4OTJiMmJmY2NhYjBhODhjY2EyYTRiMGQyMjMzOWFkMDhkOWZiNzhhZWEwZTY4ZjhmNmNhMGNjNTUyYjIwZDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SsQB12ObAB-Ubs43fsxL2lYBOMqhYL1kVUtWvPXkaJw)

The styling changed for every organizsations. I saved the styling using another email id with one organization. Then I login using another email with different organization and it changed for every organization

**Comment by CyrineKamoun** [comment]
**Project sharing bug 2.0**
-When a project is shared with an organization as viewer, all members including project owners become viewers.
- When sharing a project, we can't go from viewer to editor (or editor to viewer):  one should first remove the access for everyone and then change it to viewer/editor

**Comment by nihar1024** [comment]
> > Erro while downlowding databse (csv or excel) that I imported earlier in a project
> 
> This seems to work for me. Would you please be able to send me one of the files you're testing with (on Teams) and I can take a look?

Thanks for reporting this Cyrine. It's now been fixed and will be deployed shortly.

**Comment by gokulgpn** [comment]
**Address Pinpoint** I want to navigate to an address, however it shows only the area of the address not pinpoint to the exact location, which makes me hard to locate as well as do some scenario analysis
![Screenshot 2024-09-19 112210](https://github.com/user-attachments/assets/68165c84-e24d-4690-a482-e6e45e6e25b3)

**Comment by CyrineKamoun** [comment]
![image](https://github.com/user-attachments/assets/03e2e570-9966-48bb-9afd-492a7a70f7dd)
i reloaded the page while the job is running. When doing that the required layer (for the buffer) showed up but the job status is still ongoing

**Comment by nihar1024** [comment]
> **Error Upload** Getting error when uploading csv, gpkg files

Thanks Gokul, this is now fixed. Will be deployed shortly.

**Comment by CyrineKamoun** [comment]
> **Project sharing bug 2.0** -When a project is shared with an organization as viewer, all members including project owners become viewers.
> 
> * When sharing a project, we can't go from viewer to editor (or editor to viewer):  one should first remove the access for everyone and then change it to viewer/editor

it does not seem to be possible to have both access right for a team and an organisation simultaneously on one project

**Comment by gokulgpn** [comment]
> rby or catchement area

Catchment Area PT Transport not working

**Comment by CyrineKamoun** [comment]
> > rby or catchement area
> 
> Catchment Area PT Transport not working

it worked for me for Munich but not in France for example (bc the starting point was not in the geofence)

**Comment by gokulgpn** [comment]
> > I am unable to view the data after filter is being applied
> 
> Which dataset ?

![Screenshot 2024-09-19 120250](https://github.com/user-attachments/assets/8534f7c6-2e46-4bc7-88aa-3e40140c78b1)
it happens like this and i cannot see the data after applying filter. POI Gesundheit

**Comment by CyrineKamoun** [comment]
"Error removing members"
Error message obtained when trying to remove a member from a team that has just been created

**Comment by CyrineKamoun** [comment]
When deleting a project that was shared by another member (shared to the organization), I get a success message but in fact the project is not deleted

**Comment by CyrineKamoun** [comment]
![image](https://github.com/user-attachments/assets/9cb46ae3-6743-4fda-9935-e87d0bc9095d)
No database appearing in the catalog for Land-use (even though there seems to be 2 databases)

**Comment by CyrineKamoun** [comment]
When surfing on the catalog (on the main page), I can go, for example, to page 2. Then if I filter the data category to land use (having only two databases), then I can't access this database, nor can I go back to page 1 to see them. I have to go back to  a selection that contains several pages, select page 1, and then I can see the databases when selecting the land category
==> the interface should maybe automatically go back to page 1 or provide button to move to previous pages

**Comment by CyrineKamoun** [comment]
**Usage and quota*
"Quota units for projects are used each time a user creates a new project on the GOAT Platform. This represents the total number of projects that can be created by all users within the organization."

Deleting a project is not supposed to reduce the quota unit by 1 (from 5 to 4 for example). It is the case now.

**Comment by gokulgpn** [comment]
**Output Layer to QGIS** I downloaded the buffered polygon(with union and difference active) and imported to QGIS. However I found that I am unable to edit the layer, though spatial operations work. I did an error check and and found that self intersection error persists for the layer. I have to use Repair shapefile, then only I can edit the layer. The same happens with other output of the accessibility indicators

**Comment by EPajares** [comment]
> *_Usage and quota_ "Quota units for projects are used each time a user creates a new project on the GOAT Platform. This represents the total number of projects that can be created by all users within the organization."
> 
> Deleting a project is not supposed to reduce the quota unit by 1 (from 5 to 4 for example)? It is the case now.

Yes the number of projects should decrease if I delete one. Is this not the case? I tested it for me it works.

**Comment by CyrineKamoun** [comment]
> > *_Usage and quota_ "Quota units for projects are used each time a user creates a new project on the GOAT Platform. This represents the total number of projects that can be created by all users within the organization."
> > Deleting a project is not supposed to reduce the quota unit by 1 (from 5 to 4 for example)? It is the case now.
> 
> Yes the number of projects should decrease if I delete one. Is this not the case? I tested it for me it works.

Yes it works then, I was not sure about the functionality of the project quota

**Comment by EPajares** [comment]
> ![image](https://github.com/user-attachments/assets/0da07e2b-ee8a-4908-b8f0-32d4aeee3d77) Goat does not allow large database analysis: is it credit issues or computation error?

Maybe this has to be clearer in the docs. We can organize a docs review next week to improve things further there.

**Comment by gokulgpn** [comment]
**Data Catalog Bug** I navigated to Catalog session and selected one dataset. Instead of showing the dataset, it automatically navigates to the the Dataset section and show nothing

**Comment by gokulgpn** [comment]
> > **Error Upload** Getting error when uploading csv, gpkg files
> 
> Thanks Gokul, this is now fixed. Will be deployed shortly.

I think I am still getting error when uploading csv

**Comment by gokulgpn** [comment]
**Basemap issue** I changed basemap to dark and my team member is not abl eto see the changes. He sees all the time high fidelity and needs to manually change it

**Comment by gokulgpn** [comment]
** Viewer download project  data**  I dont want the viewer member in my organization to download the project data. Is there a way to do that?

**Comment by gokulgpn** [comment]
![Screenshot 2024-09-22 160955](https://github.com/user-attachments/assets/3ca3bbe3-6a33-4d5b-b67c-3fc1f4b3c14d)
i cannot see my layers

**Comment by CyrineKamoun** [comment]
when clicking on the "your trial expire in xx days" in the goat.plan4better version, we are redirected to plan4better.de/contact (in the url bar) but we there is a 404 not found message appearing

**Comment by CyrineKamoun** [comment]
![image](https://github.com/user-attachments/assets/4807f49f-6752-4e98-accb-77a37844edc5)
when clicking in the register for Demo > 404 not found page

**Comment by CyrineKamoun** [comment]
After changing the setting of my goat account into german, the documentation page still opens up in English

**Comment by gokulgpn** [comment]
![Screenshot 2024-09-27 132641](https://github.com/user-attachments/assets/ae53902c-6fa7-4acb-a639-4d6ddc5cd4a8)
are we able to see these datas?

**Comment by SmithU64** [comment]
It does not seem do be possible to turn on Strokes (Outlines) for a Feature Layer when the fill is turned off. I ran into it with the "Stadtteile Deutschland" Layer.

**Comment by EPajares** [comment]
> ![Screenshot 2024-09-27 132641](https://private-user-images.githubusercontent.com/99094073/371513602-ae53902c-6fa7-4acb-a639-4d6ddc5cd4a8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0NDkwMTUsIm5iZiI6MTcyNzQ0ODcxNSwicGF0aCI6Ii85OTA5NDA3My8zNzE1MTM2MDItYWU1MzkwMmMtNmZhNy00YWNiLWE2MzktNGQ2ZGRjNWNkNGE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI3VDE0NTE1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjNjRmYzE0YjExZjY5YjZlNTgzNTE4NGZmZjY3MWEwNzFhNjVhMzcyYzEyNDQ4Njk3NzY2MWU0Y2UyMzg3NGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.AUOO2R3Y78GqGgoTYS1w0TcAlLXr5F27dwlFLNk4Njs) are we able to see these datas?

data show when loading the layer. It is just the thumbnail.

**Comment by SmithU64** [comment]
After running PT Nearby Station Access Analysis the Marker Size of the symbols in the output layer can't be adjusted. When dragging the slider Marker Size the symbol-size changes for a second and than jumps back to its original size.

**Comment by CyrineKamoun** [comment]
We receive two emails (with similar content in 15 minutes time interval) when the trial period is ending suggesting contacting plan4better

**Comment by gokulgpn** [comment]
**Deleting Layers** Is there a way to delete all the layers by selecting and removing like in QGIS? I find it difficult to delete individual layers!

**Comment by nihar1024** [comment]
This issue thread is now closed!

Please report any new issues by going to https://github.com/goat-community/goat/issues > "New Issue" and labelling the issue as a `Bug`.
